### DISCUSSION TITLE
Function optimization affecting use

### DISCUSSION THREAD

**Discussion by lzb-libra** [dbeaver/dbeaver#18722] [Q&A] [answered]
1. I hope to add the shortcut key customization function. Some default shortcut keys are not easy to use
2. I hope that every time I install a new version, I will not be asked to reinstall the plug-ins that I have already installed. I have encountered this problem in previous versions. I don't know whether the new version has been optimized.

**Comment by E1izabeth** [comment, marked as answer]
Hi, @lzb-libra 
1. You can change shortcuts in Preferences -> User Interface -> Keys
![image](https://user-images.githubusercontent.com/28875055/211397675-c5372ec5-07f4-4114-87f1-061775456956.png)

Also you can open a hint with shortcuts by `Ctrl + Shift + L` (Show Key Assist)
![image](https://user-images.githubusercontent.com/28875055/211397661-cbc1b6b9-2462-4a3b-98a1-4e5044e15e38.png)

Does it satisfy your needs?

2. It's a known issue #5317

**Comment by lzb-libra** [comment]
@E1izabeth Thank you very much. I didn't check the settings carefully. This is the function I need.

**Comment by lzb-libra** [comment]
@E1izabeth Thank you very much. I didn't check the settings carefully. This is the function I need.

**Comment by lzb-libra** [comment]
Today, dbeaver informed me to update the software. After the update, I found that the plug-in I originally installed disappeared again, and I clicked the Export Data button and an error prompt popped up
![QQ截图20230110102535](https://user-images.githubusercontent.com/63389341/211448103-60a023cc-93d3-427e-bc66-65ef25538496.png)
